protocol/testing/containertest/containertest.sh

```diff
@@ -7,6 +7,7 @@ set -eo pipefail
 source "./genesis.sh"
 
 CHAIN_ID="localdydxprotocol"
+PREUPGRADE_VERSION="v2.0.0"
```

Comment:
Wondering how we will manage this version string? As of now wre we thinking just manually updated this for every new major release?

Reply:
Yup those two will need to be changed. Probably in the same PR as adding new upgrade handler